templates/download/index.html

```diff
@@ -110,7 +110,7 @@ <h3>
         </a>
       </h3>
       <p>
-        Ubuntu is the reference OS for OpenStack. Try Canonical's OpenStack on a single machine or start building a production cloud on a cluster &mdash; just add servers.
+        Ubuntu is the reference OS for OpenStack. Try <a href='/openstack/install'>Canonical OpenStack</a> on a single machine or start building a production cloud on a cluster &mdash; just add servers.
```

Comment:
I think in HTML the convention is to use double quotes instead of single quotes.

Reply:
Thanks Carlos for reviewing! Fixed!